modules/TestUtils.js

```diff
@@ -28,6 +28,12 @@ export const isArray = (object) =>
   Array.isArray(object)
 
 /**
+ * Returns true if the given object is a set.
+ */
+export const isSet = (object) =>
+  object instanceof Set
```

Comment:
this check is unreliable across realms (like iframes). `instanceof` shouldn't be used.